pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -203,12 +211,27 @@ func (dswp *desiredStateOfWorldPopulator) findAndAddNewPods() {
 	}
 
 	processedVolumesForFSResize := sets.NewString()
+	skippedDueToPVCsUnderDeletion := false
 	for _, pod := range dswp.podManager.GetPods() {
 		if dswp.isPodTerminated(pod) {
 			// Do not (re)add volumes for terminated pods
 			continue
 		}
-		dswp.processPodVolumes(pod, mountedVolumesForPod, processedVolumesForFSResize)
+		if dswp.processPodVolumes(pod, mountedVolumesForPod, processedVolumesForFSResize) {
+			skippedDueToPVCsUnderDeletion = true
```

Comment:
processPodVolumes can return skipped due to other reasons

Reply:
This value is to pass the return from processPodVolumes to test in order to assert the cause for skipping.